projects/aca-folder-rules/src/lib/services/folder-rules.service.ts

```diff
@@ -174,13 +174,17 @@ export class FolderRulesService {
   }
 
   loadAspects(): void {
```

Comment:
This is more relevant to the actions service since it calls the `action-parameter-constraints` endpoint. Could you move it there?

Reply:
Moved.